.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Description
+<!--- Describe your changes in detail -->
+
+### Ticket
+<!-- Add link to the corresponding ticket -->
+
+### Screenshots
+
+| Before | After |
+| ------ | ----- |
+|        |       |
+
+### How do I test this code?
+<!-- Describe how a reviewer can run and test this new functionality.  What is the expected behaviour?  -->
+
+### Reminder
+Have you completed these common tasks? (remove those that don't apply)
+
+- [ ] **Documentation** updated or created
+- [ ] **Tests** written for new or updated for existing functionality
```

Comment:
```suggestion
- [ ] **Tests** written for new (or updated for existing) functionality
```
**suggestion (non-blocking)**: it wasn't you that wrote the original; I think this sentence reads a little better with brackets.